packages/test/src/integration-tests.ts

```diff
@@ -1207,6 +1211,68 @@ export function runIntegrationTests(codec?: PayloadCodec): void {
     at stackTracer (test/src/workflows/stack-tracer.ts)`
       );
     });
+
+    test('Enhanced stack trace returns trace that makes sense', async (t) => {
+      const { client } = t.context;
+      const workflowId = uuid4();
+
+      const enhancedStack = await client.execute(workflows.enhancedStackTracer, {
+        taskQueue: 'test',
+        workflowId,
+      });
+
+      const stacks = enhancedStack.stacks.map((s) => ({
+        locations: s.locations.map((l) => ({
+          ...l,
+          ...(l.filePath ? { filePath: l.filePath.replace(path.resolve(__dirname, '../../../'), '') } : undefined),
+        })),
+      }));
+      t.is(enhancedStack.sdk.name, 'typescript');
+      t.is(enhancedStack.sdk.version, pkg.version); // Expect workflow and worker versions to match
+      t.deepEqual(stacks, [
+        {
+          locations: [
+            {
+              functionName: 'Function.all',
+            },
+            {
+              filePath: '/packages/test/src/workflows/stack-tracer.ts',
+              functionName: 'enhancedStackTracer',
+              line: 32,
```

Comment:
Validating that line and column fields are set seems like a good thing, but having actual line number and column numbers here will make this test pretty fragile. Not sure how to make it better though.

Could we simply confirm that both fields are integer numbers greater than 1?
Or round the numbers to the next multiple of 20? Not pretty, but it would at least means that the values would rarely change...

Reply:
I did remove the line numbers in the basic stack trace test, I guess it's fine to leave them here.
I don't expect this file to change much.
Don't mind removing them either.

Reply:
No strong opinion on this.